### PR TITLE
Update trgt to 1.0.0

### PR DESCRIPTION
**Pull request by tmokveld**
Update trgt to 1.0.0. Added corresponding changes to manage consolidation of trgt and trvz binaries.

## Summary by CodeRabbit

- **New Features**
	- Updated `trgt` package to version 1.0.0 with a new source URL format.
	- Added a new license for the `trgt` package.
	- Included an additional recipe maintainer.

- **Bug Fixes**
	- Removed outdated source entry and adjusted test commands for clarity.

- **Chores**
	- Deleted the `build.sh` script, impacting the automated setup process for binaries.

### CODE REVIEW

File: recipes/trgt/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
   summary: Tandem repeat genotyping and visualization from PacBio HiFi data
 
-extra:
-  recipe-maintainers:
-    - zeeev
-    - williamrowell
-
 build:
   number: 0
   noarch: generic
```
Comment (martin-g) [inline recipes/trgt/meta.yaml:20]: The downloaded tarball says linux-64 but here it says noarch. IMO this line should be removed

**Comment by tmokveld** [comment]
Running the linter locally after this change, I see:
```
ERROR: recipes/trgt/meta.yaml:22: should_be_noarch_generic: The recipe should be build as `noarch`
```
Which seems somewhat contradictory. What is the recommended fix?

**Comment by tmokveld** [comment]
@BiocondaBot please add label

File: recipes/trgt/meta.yaml [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
-    - williamrowell
-
 build:
   number: 0
-  noarch: generic
+  skip: true  # [not linux64]
+  script:
+    - mkdir -p ${PREFIX}/bin
+    - cp trgt ${PREFIX}/bin/
+    - chmod +x ${PREFIX}/bin/trgt
```
[the remark this answers is not in the document]
Comment (tmokveld) [reply recipes/trgt/meta.yaml]: @martin-g Is there any other suggested change to fix this? I seem to be running in circles with these changes...

[the remark this answers is not in the document]
Comment (martin-g) [reply recipes/trgt/meta.yaml]: Yes, sorry about that! It seems the Bioconda build does not support it - https://github.com/bioconda/bioconda-recipes/issues/50985#issuecomment-2378842986
Please use the lint skip for now!

[the remark this answers is not in the document]
Comment (tmokveld) [reply recipes/trgt/meta.yaml]: Thanks, I'll do that!

**Review by martin-g — APPROVED** [review]

**Pull request merged**